src/js/Trigger.js

```diff
@@ -39,34 +53,54 @@ export default class Trigger {
   }
 
   _bindEvents() {
-    this.settings.el.addEventListener('mouseenter', this._handleEntry, false);
-    this.settings.el.addEventListener('mouseleave', this._hide, false);
-    this.settings.el.addEventListener('mousemove', this._handleMovement, false);
+    this.settings.el.addEventListener("mouseenter", this._handleEntry, false);
+    this.settings.el.addEventListener("mouseleave", this._hide, false);
+    this.settings.el.addEventListener("mousemove", this._handleMovement, false);
 
     if (this.settings.handleTouch) {
-      this.settings.el.addEventListener('touchstart', this._handleEntry, false);
-      this.settings.el.addEventListener('touchend', this._hide, false);
-      this.settings.el.addEventListener('touchmove', this._handleMovement, false);
+      this.settings.el.addEventListener("touchstart", this._handleEntry, false);
+      this.settings.el.addEventListener("touchend", this._hide, false);
+      this.settings.el.addEventListener(
```

Comment:
Seems like it's choosing to line-break this call because it exceeds some arbitrary line-length limit, but it really just makes it hard to read IMO. Any chance we can disable this behavior?

Reply:
There's two options here - we can change the line length limit (120 is another common setting instead of the current 80), or we can add the comment `// prettier-ignore` to the line above, which will ignore formatting of the next line.

Reply:
Let's change it to 120, then. If we get to the point where we have to regularly add comments to turn off a tool that's supposed to be helping us I'll probably end up throwing my computer out a window 😶.